doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -740,112 +738,31 @@ Induction
          Goal forall l:list nat, contains0 (1 :: l) -> contains0 l.
          intros l H; inversion H as [ | l' p Hl' [Heqp Heql'] ].
 
-.. tacv:: inversion @natural as @or_and_intropattern
-
-   This allows naming the hypotheses introduced by :n:`inversion @natural` in the
-   context.
-
-.. tacv:: inversion_clear @ident as @or_and_intropattern
-
-   This allows naming the hypotheses introduced by ``inversion_clear`` in the
-   context. Notice that hypothesis names can be provided as if ``inversion``
-   were called, even though the ``inversion_clear`` will eventually erase the
-   hypotheses.
-
-.. tacv:: inversion @ident in {+ @ident}
-
-   Let :n:`{+ @ident}` be identifiers in the local context. This tactic behaves as
-   generalizing :n:`{+ @ident}`, and then performing ``inversion``.
-
-.. tacv:: inversion @ident as @or_and_intropattern in {+ @ident}
-
-   This allows naming the hypotheses introduced in the context by
-   :n:`inversion @ident in {+ @ident}`.
-
-.. tacv:: inversion_clear @ident in {+ @ident}
+.. tacn:: inversion_clear {| @ident | @natural } {? as @or_and_intropattern } {? in {+ @ident } }
```

Comment:
Can we indent (to make them appear as variants) everything from `inversion_clear` to `simple inversion`? Also, I suggest moving `simple inversion` up just after `inversion_clear`, and using the same order for the `dependent inversion` variants, which could all be gathered in a single `.. tacn::`.

Reply:
> which could all be gathered in a single `.. tacn::`.

You mean like this?? I don't see the benefit.  The descriptions are different.
```
   .. tacn:: dependent inversion {| @ident | @natural } {? as @or_and_intropattern } {? with @one_term }
             dependent inversion_clear {| @ident | @natural } {? as @or_and_intropattern } {? with @one_term }
             dependent simple inversion {| @ident | @natural } {? as @or_and_intropattern } {? with @one_term }
```

Reply:
No, not like this. Just indenting and reordering the tactics and their descriptions, not gathering the descriptions together.